src/dsp_tools/commands/xmlupload/prepare_xml_input/prepare_xml_input.py

```diff
@@ -165,6 +169,25 @@ def _extract_permissions_from_xml(root: etree._Element, proj_context: ProjectCon
     return {permission.permission_id: permission for permission in permissions}
 
 
+def _extract_authorships_from_xml(root: etree._Element) -> dict[str, list[str]]:
+    # The xsd file ensures that the body of the element contains valid non-whitespace characters
+    def get_one_author(ele: etree._Element) -> str:
+        txt = cast(str, ele.text)
+        fragment_list: list[str] = []
+        for frag in txt.split(" "):
+            no_newline = frag.split("\n")
+            for sub_frag in no_newline:
+                fragment_list.extend(sub_frag.split("\t"))
+        split_auth = [found for x in fragment_list if (found := x.strip())]
+        return " ".join(split_auth)
```

Comment:
This is a bit difficult to understand, and can be achieved easier (at least, my version passes your unit test):

```suggestion
        txt = cast(str, ele.text)
        txt = re.sub(r"[\n\t]", " ", txt)
        txt = re.sub(r" +", " ", txt)
        return txt.strip()
```